include/random_walks/gaussian_accelerated_billiard_walk.hpp

```diff
@@ -148,6 +163,12 @@ struct GaussianAcceleratedBilliardWalk
             int it;
             NT coef = 1.0;
             NT vEv;
+            typename Point::Coeff b;
+            NT* b_data;
+            if constexpr (std::is_same<MT, Eigen::SparseMatrix<NT, Eigen::RowMajor>>::value) {
```

Comment:
if this is not the case then `b_data` is unitialized! Isn't is cleaner if you define and initialize it just before its use, below?